cmd/kube-controller-manager/app/batch.go

```diff
@@ -46,6 +47,17 @@ func startCronJobController(ctx ControllerContext) (http.Handler, bool, error) {
 	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "batch", Version: "v1beta1", Resource: "cronjobs"}] {
 		return nil, false, nil
 	}
+	if features.DefaultMutableFeatureGate.Enabled(features.CronjobController2) {
+		cj2c, err := cronjob.NewController2(ctx.InformerFactory.Batch().V1().Jobs(),
+			ctx.InformerFactory.Batch().V1beta1().CronJobs(),
+			ctx.ClientBuilder.ClientOrDie("cronjob-controller"),
```

Comment:
nit: cronjob-controllerv2 to make it explicit in the metrics.

Reply:
+1, modified

Reply:
> For other readers, we went with the same name, so that we can re-use the SA for this controller.

Please create a new SA and tie the existing rolebinding to it so we can figure out what is being used.  This can be a followup (still in 1.120) to avoid making this PR larger.